app/assets/stylesheets/blocks/_parser_templates.scss

```diff
@@ -1,27 +1,10 @@
 
 .parser_templates.edit {
 	.button-group {
-		margin-top: 20px; 
-	}
-	#hidden-parser-template-form {
-		display: none;
+		margin-top: 20px;
```

Comment:
seems like we are using a mix of ems and pxs.

Reply:
Yeah the manager definitely needs some design love ^^